cfgov/jinja2/v1/owning-a-home/_templates/brand-footer.html

```diff
@@ -0,0 +1,85 @@
+
+{# ==========================================================================
+
+   render(links)
+
+   ==========================================================================
+
+   Description:
+
+   Renders all the brand footer content: 3 link sections and email section.
+
+   links: An array of three link objects, each containing:
+            - text (req): text of link section
+            - img: img for link section
+            - label: link label
+            - url: link url
+
+            Macro will display max of three link sections. If there are
+            more than three link objects, only the first three will
+            be shown. If no links are passed in, only the email signup
+            will be shown.
+   ========================================================================== #}
+
+{% macro render(links=[]) %}
+<footer aria-label="Owning a Home footer" class="brand-footer content">
+    <div class="content_wrapper">
+        <h3 class="brand-footer_headline u-mb0">
```

Comment:
Is `brand-footer_headline` used elsewhere? Can we just set `margin-bottom: 0` in its CSS instead of having the utility class here.